app/models/trip.rb

```diff
@@ -0,0 +1,6 @@
+class Trip < ApplicationRecord
+  belongs_to :user
+  has_one :vehicle
+  serialize :destinations, Array
+
```

Comment:
add some validations, for example destination.size must be >= 2. origin and destinations must be different